src/frontend/modules/twitter/widgets/tweets.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Frontend\Modules\Twitter\Widgets;
+
+use Frontend\Core\Engine\Model as FrontendModel;
+use Frontend\Core\Engine\Base\Widget as FrontendBaseWidget;
+use Frontend\Modules\Twitter\Engine\Model;
+
+/**
+ * This is the tweets widget.
+ *
+ * @author Jonas De Keukelaere <jonas@sumocoders.be>
+ * @author Toon Daelman <toon@sumocoders.be>
+ */
+class Tweets extends FrontendBaseWidget
+{
+    /**
+     * Execute the extra
+     */
+    public function execute()
+    {
+        parent::execute();
+        $this->loadTemplate();
+        try {
+            $this->parse();
+        } catch (\Exception $e) {
```

Comment:
Er zit een SpoonTemplateException in en je hebt in die twitter library een heleboel \Exception 's